### PR TITLE
WDT-XXX Add modules

### PR DESCRIPTION
**Pull request by miguelmatosduarte**
Add go modules

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module github.com/miguelmatosduarte/instrumentedsql
+
+go 1.11
```
Comment (TheJoas) [inline go.mod]: Can we bump this up or there is a reason for this? (just curious)

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module github.com/miguelmatosduarte/instrumentedsql
```
Comment (TheJoas) [inline go.mod]: Shouldn't this be module `github.com/pcruz7/instrumentedsql`?

Comment (miguelmatosduarte) [reply go.mod]: it should, fixed :)

File: go.mod (hunk 1 above)
Comment (miguelmatosduarte) [reply go.mod]: yeah we can bump it, no problem :) 1.17 alright?

Comment (TheJoas) [reply go.mod]: deal

**Review by TheJoas — APPROVED** [review]

**Pull request merged**